utils/lib.sh

```diff
@@ -725,6 +725,16 @@ pyenv.cmd() {
     )
 }
 
+
+pyenv.activate() {
+	pyenv.install
+    # shellcheck source=/dev/null
+	source "${PY_ENV_BIN}/activate"
+    # make sure the exit code is 0, this function can be called with set -e
+	[ "$VERBOSE" = "1" ] && set -x || :
```

Comment:
Since this is not a subshell, do not activate `set -x` in a function.  The caller should decide if he wants to log `set -x`

IMO the following should be enough ...

```sh
pyenv.activate() {
    pyenv.install
    # shellcheck source=/dev/null
    source "${PY_ENV_BIN}/activate"
}
```